pyproject.toml

```diff
@@ -0,0 +1,50 @@
+[tool.poetry]
+name = "opendevin"
+version = "0.1.0"
+description = "OpenDevin: Code Less, Make More"
+authors = ["OpenDevin"]
+license = "MIT"
+readme = "README.md"
+repository = "https://github.com/OpenDevin/OpenDevin"
+
+[tool.poetry.dependencies]
+python = "^3.11"
+datasets = "*"
+pandas = "*"
+litellm = "*"
+termcolor = "*"
+torch = {version = "*", source = "pytorch"}
```

Comment:
is there a reason this is not just `torch = "*"`

Reply:
this version is torch cpu one I'not sure why they did it but it was from the previous Pipfile